presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/clientproviders/RetryingPrestoThriftServiceProvider.java

```diff
@@ -96,11 +101,13 @@ public PrestoThriftService selectedHostClient(List<HostAddress> hosts)
         private final Supplier<PrestoThriftService> clientSupplier;
         private final RetryDriver retry;
         private PrestoThriftService client;
+        private RetryingPrestoThriftServiceStats retryingPrestoThriftServiceStats;
 
-        public RetryingService(Supplier<PrestoThriftService> clientSupplier, RetryDriver retry)
+        public RetryingService(Supplier<PrestoThriftService> clientSupplier, RetryDriver retry, RetryingPrestoThriftServiceStats retryingPrestoThriftServiceStats)
```

Comment:
In the caller this parameter is called just `stats`. Let's call it the same here.